EMS/core-bundle/src/Core/Submission/ExportConfig.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EMS\CoreBundle\Core\Submission;
+
+use EMS\Helpers\Standard\Json;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+final readonly class ExportConfig
+{
+    /**
+     * @param mixed[]  $columns
+     * @param string[] $emailsTo
+     */
+    public function __construct(
+        public array $columns,
+        public array $emailsTo,
+        public string $subject,
+        public ?string $filter = null,
+        public ?string $filename = 'crm-export',
+        public ?string $format = 'xlsx',
```

Comment:
As static parameters?

Reply:
(In fact, the class is already readonly)